docs/lido-dao.md

```diff
@@ -1,17 +1,46 @@
+The DAO will accumulate service fees from Lido, which can be funnelled into the insurance and development funds, distributed by the DAO.
 
-## LDO Token
+## Functions
 
-LDO is a [MiniMeToken](https://github.com/Giveth/minime) granting governance rights in the Lido DAO. By holding the LDO token, one is granted voting rights within the Lido DAO. The more LDO locked in a user’s voting contract, the greater the decision-making power the voter gets.
+Lido is managed by the Lido DAO. The DAO members govern Lido to ensure its efficiency and stability. The Lido DAO should do the following:
+- Build, deploy update and decide on key parameters of liquid staking protocols, appprove incentives for parties that contribute towards DAO’s goals
+- Node operators management. Assign initial DAO-vetted node operators, scout and qualify new node operators and penalise the existing ones slashed by chains rules
+- Approve LEGO grants to support different research and so initiatives protocol guilds
+- Payments to full-time contributors and other operational duties
+- Security, bug bounty program, respond to emergency
+- Accumulation of service fees from Lido, which can be funnelled into the insurance and development funds, distributed by the DAO.
+
+
+## Governance
+
+The `LDO` token governs all Lido DAO governance and network decisions to ensure its prolonged stability and decentralised decision-making to facilitate the growth of fair, trustless, and transparent liquid staking.  The `LDO` contract address - `0x5a98fcbea516cf06857215779fd812ca3bef1b32`.
+
+> 📝 For the more detailed information about governance, please, check out [governance](https://lido.fi/governance) page. 
+
+Generally speaking, the `LDO` governance token is allowing to distribute all decision-making to create a governance minimised staking service built around community-growth and self-sustainability. To have a vote in the Lido DAO, and to contribute to the determination of any of the topics outlined above, one must hold the `LDO` governance token. Holding `LDO` gives DAO members a vote in the future of Lido, allowing each DAO member to have a personal say in the community. `LDO` voting weight is proportional to the amount of `LDO` a voter stakes in the voting contract. The more LDO in a user’s voting contract, the greater the decision-making power the voter gets. The exact mechanism of `LDO` voting can be upgraded just like the other DAO applications.
+
+> 📝 If you have any initiatives you think will benefit the Lido protocol, share your thoughts in our [governance forum](https://research.lido.fi).
+
+## Software
+
+The Lido DAO is an [Aragon](https://aragon.org/dao) organization. Since Aragon provides a full end-to-end framework to build DAOs, we use its standard tools.
+
+> 📝 The governance process only takes place within the Ethereum network. For other networks, this process is implemented through committee and multisig (we need a multisig list).
```

Comment:
Probably, need some refs to multisig explanation and how it can be used to govern a protocol.

Reply:
Could you please provide me with them? Everyone I spoke to has no idea where to get the links.